Cargo.toml

```
@@ -61,7 +61,7 @@ expect-test = "1"
 png = "0.17"
 proptest = "1.4"
 rstest = "0.18"
-sspi = "0.13"
+sspi = "0.14.1"
```

suggestion: Do not specify the patch version to avoid over-constraining. It’s technically correct here since we don’t really want to accept 0.14.0, but dependabot will keep the patch number around forever otherwise. 
```suggestion
sspi = "0.14"
```